### PR TITLE
Hi! I have made some modifications to layout problems.

### PR DESCRIPTION
**Pull request by yuhuitech**
Dear Developers!

I am the creator of issue #31, and I have made some modifications to this problem. Now the permission page is scrollable now. I would appreciate it if you could kindly revise my code and leave me some feedback. Thank you so much for your precious time!

Here is the screenshot after my modification:

copy

and before my modification:

copy

### CODE REVIEW

File: smartautoclicker/src/main/res/layout/dialog_permissions.xml [hunk 1]
```diff
@@ -15,6 +15,11 @@
...
  * along with this program; If not, see <http://www.gnu.org/licenses/>.
 -->
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
```
Comment (kevin-tagheuer) [inline smartautoclicker/src/main/res/layout/dialog_permissions.xml:25]: xmlns elements should always be declared on the first element, so:
* xmlns:android should be removed from here as you declared it above
* xmlns:tools should be moved in the ScrollView element

File: smartautoclicker/src/main/res/layout/dialog_permissions.xml [hunk 2]
```diff
@@ -15,6 +15,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; If not, see <http://www.gnu.org/licenses/>.
 -->
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
```
Comment (kevin-tagheuer) [inline smartautoclicker/src/main/res/layout/dialog_permissions.xml:22]: You need to correct the indent on the elements inside your ScrollView element.
Simply add 4 spaces to the LinearLayout and its content

**Review by kevin-tagheuer — COMMENTED** [review]
The commit message should be explaining the content of the commit.
Here, something like "Add scrolling to the permission dialog" will be good.

Otherwise, thanks for contributing !

File: smartautoclicker/src/main/res/layout/dialog_permissions.xml (hunk 1 above)
Comment (yuhuitech) [reply smartautoclicker/src/main/res/layout/dialog_permissions.xml:20]: OK, thank you so much for your advice, and I have changed this part according to your comments.

File: smartautoclicker/src/main/res/layout/dialog_permissions.xml (hunk 2 above)
Comment (yuhuitech) [reply smartautoclicker/src/main/res/layout/dialog_permissions.xml]: I have changed the indent in this file. :)

**Review by Nain57 — APPROVED** [review]

**Pull request merged**